packages/blockchain-wallet-v4-frontend/src/data/components/uploadDocuments/actions.js

```diff
@@ -1,33 +1,33 @@
 import * as AT from './actionTypes'
 
-export const fetchData = token => ({
-  type: AT.FETCH_DATA,
-  payload: { token }
+export const fetchData = (token) => ({
```

Comment:
Was upload documents converted to redux toolkit? If so I think this file can be deleted or is it still used in the old redux reducer file?

Reply:
ah no I actually used different one not this one since this is something non interest related, this change is a result of auto fix of format on file open/save